server.js

```diff
@@ -22,12 +13,60 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+// Start defining the routes
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Books API')
+})
+
+// Return all the books (objects) from the array in books.json
+app.get('/books', (req, res) => {
+  console.log(books)
+  res.json(books)
+})
+
+// Return the first half of the books (objects) from the array in books.json
+app.get('/books1', (req, res) => {
+  console.log(books.slice(0, 251))
+  res.json(books.slice(0, 251))
+})
+
+// Return the second half of books (objects) from the array in books.json
+app.get('/books2', (req, res) => {
+  console.log(books.slice(251, 500))
+  res.json(books.slice(251, 500))
+})
```

Comment:
It would be nice to see these endpoints tied into the `/books` endpoint, and use pagination to filter the number of results returned.